### ISSUE TITLE
🐛 run at flutter web

### ISSUE THREAD

**Issue by longfit** [bdlukaa/fluent_ui#999]
Hi
I have add login page similar go_router case.
I can run at Debgger or flutter run -d chrome 

cannot run Builded flutter web app after deployment at  web server
or 
I cannot run at python3 -m http.server 8088 at <project_dir>/build/web
python3 shown
127.0.0.1 - - [23/Dec/2023 21:41:58] "GET /assets/assets/images/yst_banner_title_2019.png HTTP/1.1" 200 -
127.0.0.1 - - [23/Dec/2023 21:42:13] code 404, message File not found
127.0.0.1 - - [23/Dec/2023 21:42:13] "GET /login HTTP/1.1" 404 -
127.0.0.1 - - [23/Dec/2023 21:42:13] "GET /manifest.json HTTP/1.1" 304 -
==========================================================

review chrome devtool

when I type url http://192.168.3.207/
the login page load and shown but login no response,
devtools shown error :
flutter.js:362 Exception while loading service worker: Error: Service Worker API unavailable.
The current context is NOT secure.
...
attached the screen and

expect screen result

![Screenshot from 2023-12-23 20-26-16](https://github.com/bdlukaa/fluent_ui/assets/4157794/73038ff5-a613-426a-95fe-7efcafb0a1fb)

Basically , after I add login page based go_router redirection.dart example,
I can work properly linix, windows, android  ios
except web.
since go_router example show url localhost:99999/#/login 
but fluent_ui show localhost:9999/login

[Screencast from 2023-12-23 20-33-51.webm](https://github.com/bdlukaa/fluent_ui/assets/4157794/7788cecf-c0f4-45ec-9f8c-13e26fe6a69e)

I am not sure fluent_ui  run at flutter web caused or misuse go_router

**Comment by bdlukaa** [comment]
You must configure your url strategy: https://docs.flutter.dev/ui/navigation/url-strategies.

# is the hashed strategy, while no # is the path strategy!

**Comment by longfit** [comment]
Hi

no matter use
 setPathUrlStrategy();
or not

I cannot launch Service Worker after the deployment of the web server

but 
I can run at flutter run -d chrome 

I don't know where I was wrong after deployment
